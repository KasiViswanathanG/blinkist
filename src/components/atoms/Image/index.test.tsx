import {screen, fireEvent, getByText, render } from '@testing-library/react'
import React from 'react'
import renderer from 'react-test-renderer'

import ImageComponent from '.'


describe('Testing the ImageComponent', () => {
    it('render ImageComponent', () => {
      const image = renderer.create(<ImageComponent imgLocation={'static/images/1.png'} />).toJSON();
      expect(image).toMatchSnapshot();
    });
})  