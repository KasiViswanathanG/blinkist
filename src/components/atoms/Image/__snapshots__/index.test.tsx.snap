// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the ImageComponent render ImageComponent 1`] = `
<div
  style={
    Object {
      "display": "inline-flex",
    }
  }
>
  <img
    alt="image"
    className="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-kz0bxj-MuiCardMedia-root"
    height="140"
    src="static/images/1.png"
  />
</div>
`;
